components/Typography/typography.html

```diff
@@ -0,0 +1,228 @@
+
+                    <div class="component-display flex-col">
+                        <h1> H1 heading</h1>
+                        <h2> H2 heading</h2>
+                        <h3> H3 heading</h3>
+                        <h4> H4 heading</h4>
+                        <h5> H5 heading</h5>
+                        <h6> H6 heading</h6>
+                    </div>
+
+                    <div class="code-snippet">
+                        <code>
+                            <iframe
+                                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Ch1%253E%2520H1%2520heading%253C%252Fh1%253E%250A%253Ch2%253E%2520H2%2520heading%253C%252Fh2%253E%250A%253Ch3%253E%2520H3%2520heading%253C%252Fh3%253E%250A%253Ch4%253E%2520H4%2520heading%253C%252Fh4%253E%250A%253Ch5%253E%2520H5%2520heading%253C%252Fh5%253E%250A%253Ch6%253E%2520H6%2520heading%253C%252Fh6%253E"
+                                style="width: 100%; height: auto; border:0; transform: scale(1); overflow:hidden;"
+                                sandbox="allow-scripts allow-same-origin">
+                            </iframe>
+                        </code>
+                    </div>
+
+
+                    <h5 class="mt-5 mb-3 text-primary-color">Aligned Text </h5>
+
+                    <p class="mb-3 lh-10">To align the text add text-center, text-left, text-right class to wrapper
+                        element
+                    </p>
+
+                    <div style="display: block;" class="component-display flex-col">
+                        <h3 class="text-center">Text Align Center</h6>
+                            <h3 class="text-left">Text Align left</h6>
+                                <h3 class="text-right">Text Align right</h6>
+                    </div>
+
+                    <div class="code-snippet">
+                        <code>
+                            <iframe
+                                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%2520class%253D%2522text-center%2522%253EText%2520Align%2520Center%253C%252Fh6%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%2520class%253D%2522text-left%2522%253EText%2520Align%2520left%253C%252Fh6%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%2520class%253D%2522text-right%2522%253EText%2520Align%2520right%253C%252Fh6%253E"
+                                style="width: 100%; height: auto; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
Suggest you use class names instead of such inline styling. 
Also, these seems to be common properties in all snapshots so using class name also makes it easier to reuse and change code else we'll have to make changes in every snapshot that we've used.